react/__tests__/AccordionFilterContainer.test.js

```diff
@@ -33,4 +33,10 @@ describe('<AccordionFilterContainer />', () => {
     const { getByText } = renderComponent()
     expect(getByText('price-range1')).toBeInTheDocument()
   })
+
+  it('should match snapshot', () => {
+    const { asFragment } = renderComponent()
+
+    expect(asFragment()).toMatchSnapshot()
```

Comment:
Avoid using snapshots, prefer querying what you expect to be there

Reply:
I just realized that I didn't need an additional test since [this one](https://github.com/vtex-apps/search-result/blob/master/react/__tests__/AccordionFilterContainer.test.js#L32) already does the job.